data/hai/hai reveal 5 philosophers.txt

```diff
@@ -84,16 +85,15 @@ mission "Hai Reveal [B01-A] The Blockade Goes Up"
 # You as the player have one job: make sure everyone jumps in together, and don't die. During the first campaign it was really important the fights were all balanced for you to be able to tip the balance.
 # THIS mission is designed to give you a spectacle. A little visual demonstration of what a real battle could potentially look and feel like with low stakes for you. It's a visual feast, not a challenge.
 mission "Hai Reveal [B02] Join the Fleet"
+	autosave
 	name "Join the Fleet"
-	description "Join the fleet in <waypoint> and move on Stormhold."
-	deadline 9
-	# The minimum journey time is 7
+	description "Join the fleet and move on Stormhold. Make sure all ships have enough fuel to reach there at the same time."
+	deadline 12
+	# The minimum journey time is 7 if the player jumps system by system.
 	source "Sunracer"
-	waypoint "Sumar"
 	destination "Stormhold"
-	clearance
 	to offer
-		has "Hai Reveal [B01] Information Blackout: offered"
+		has "Hai Reveal [A14] A Time of Trouble: done"
```

Comment:
This too is a large amount of no - largely based on the actual changes needed for the previous one.

Reply:
The mission won't work without clearance, so I should not have taken that out. I was able to finish anyway since I had already dominated the planet.

Reply:
I changed how I fixed this mission.